README.md

```diff
@@ -6,24 +6,49 @@ This site is a collaborative project by the Signalen community and is meant to b
 
 ## Building, testing and deploying
 
-This site is built using [Gatsby](https://www.gatsbyjs.org/). First make sure you have [Node](https://nodejs.org/en/) installed on your machine. Then install the dependencies with the `npm install` command. After this you can use the `npm start` to start a local webserver and build/update the site.
+This site is built using [Gatsby](https://www.gatsbyjs.org/). First make sure you have [Node](https://nodejs.org/en/) installed on your machine. Then install the dependencies with the `npm install` command. After this you can use the `npm start` to start a local web server and build/update the site.
 
 The `master` branch of this repository is automatically deployed using Netlify.
 
+## How to add a news article to the website
+You can add a news article to the website by adding a Markdown document to the `pages/news/` folder. The news section of the website has multilingual support. Copy an existing file and edit it. Use the following filename convention: MM-DD-YYYY-filename.md
```

Comment:
This date format is odd. Is it possible to choose something more standardized like YYYY-MM-DD?